src/content/upgrades/beacon-chain/index.md

```diff
@@ -7,51 +7,43 @@ sidebar: true
 image: ../../../assets/upgrades/core.png
 summaryPoint1: The Beacon Chain doesn't change anything about the Ethereum we use today.
 summaryPoint2: It introduced proof-of-stake to the Ethereum ecosystem.
-summaryPoint3: It will coordinate the network, serving as the consensus layer.
-summaryPoint4: It is an essential precursor to upcoming scaling upgrades, such as sharding.
+summaryPoint3: It was merged with the original Ethereum proof-of-work chain in 2022.
+summaryPoint4: It is now handling all consensus logic and block gossip for Ethereum.
 ---
 
 <UpgradeStatus isShipped dateKey="page-upgrades-beacon-date">
-  The Beacon Chain shipped on December 1, 2020. To learn more, <a href="https://beaconscan.com/">explore the data</a>. If you want to help validate the chain, you can <a href="/staking/">stake your ETH</a>.
+  The Beacon Chain shipped on December 1, 2020. At the time it was a separate chain to Ethereum Mainnet. It ran proof-of-stake but did not handle any transactions. The Beacon Chain was merged with the original Ethereum Mainnet in 2022 to take over the consensus logic and block propagation for Ethereum. Since The Merge, there has only been one, single Ethereum chain.
 </UpgradeStatus>
 
-## What does the Beacon Chain do? {#what-does-the-beacon-chain-do}
+## What was the Beacon Chain? {#what-is-the-beacon-chain}
 
-The Beacon Chain is a ledger of accounts that conducts and coordinates the network of [stakers](/staking/). It isn't quite like the [Ethereum Mainnet](/glossary/#mainnet) of today. It does not process transactions or handle smart contract interactions.
+The Beacon Chain was the name given to the original proof-of-stake blockchain that was launched in 2020. It was created to ensure the proof-of-stake consensus logic was sound and sustainable before enabling it on Ethereum Mainnet. Therefore, it ran alongisde the original proof-of-work Ethereum. Switching off proof-of-work and switching on proof-of-stake on Ethereum required instructing the Beacon Chain to accept transactions from the original Ethereum chain, bundle them into blocks and then oprganize the blocks into a block chain using a proof-of-stake based consensus mechanism. At the same moment, the original Ethereum clients turned off their mining, block propagation and consensus logic, handing that all over to the Beacon Chain. This event was known as [The Merge](/upgrades/merge/). Once The Merge happened, there were no longer two blockchains, there was just one single proof-of-stake Ethereum.
```

Comment:
```suggestion
The Beacon Chain was the name of the original proof-of-stake blockchain that was launched in 2020. It was created to ensure the proof-of-stake consensus logic was sound and sustainable before enabling it on Ethereum Mainnet. Therefore, it ran alongside the original proof-of-work Ethereum. Switching off proof-of-work and switching on proof-of-stake on Ethereum required instructing the Beacon Chain to accept transactions from the original Ethereum chain, bundle them into blocks and then organize them into a blockchain using a proof-of-stake based consensus mechanism. At the same moment, the original Ethereum clients turned off their mining, block propagation and consensus logic, handing that all over to the Beacon Chain. This event was known as [The Merge](/upgrades/merge/). Once The Merge happened, there were no longer two blockchains; there was just one proof-of-stake Ethereum chain.
```

Reply:
Noticed we're saying 'proof-of-stake based' instead of 'proof-of-stake' in a lot of places now. Curious what the thinking behind this is?

Reply:
feedback from wider EF about definition of "consensus mechanism". The consensus mechanism is more than just the proof-of-stake part, it also includes the social coordination, fork choice rule, incentives and penalties, etc. It is hard to wrap all this into simple wordage, so we started using "pos-based mechanism" as the neatest compromise between readability and accuracy we could think of (we being Paul and I).

Reply:
Makes sense to me, was just curious :-)